.github/CODEOWNERS

```diff
@@ -0,0 +1,18 @@
+# Uses the same pattern rule for gitignore file
+# https://git-scm.com/docs/gitignore#_pattern_format
+# For more info about CODEOWNERS, visit
+# https://help.github.com/articles/about-codeowners/.
+
+#
+# These are the default owners for everything in the repo.
+#
+* @anselmbradford @cfarm @contolini @jimmynotjim @Scotchester @sebworks
+
+# To add owners of a specific file type or directory, prepend the line with
+# the type to match. The last matching pattern takes the most precedence. Ex.
+# *.js                      @js-owner
+
+#
+# These are the owners of the deployment process
+#
+/scripts/npm/prepublish/    @contolini @jimmynotjim
```

Comment:
I think we should stick with a consistent one or two spaces between pattern and users. It's weird that the global rule can't match this or its line length would be too long.

Reply:
Addressed all of these comments in the latest push.